### PR TITLE
Enable IRSA by default in release v19.0.0

### PR DESCRIPTION
**Pull request by njuettner**
Issue: https://github.com/giantswarm/roadmap/issues/1095

## Checklist

- [x] Update changelog in CHANGELOG.md.

### CODE REVIEW

File: service/controller/key/common.go [hunk 1]
```diff
@@ -180,6 +183,11 @@ func IsDeleted(getter DeletionTimestampGetter) bool {
 	return getter.GetDeletionTimestamp() != nil
 }
 
+func IsV19Release(releaseVersion *semver.Version) bool {
```
Comment (whites11) [inline service/controller/key/common.go:186]: How about checking the major version is v19 or higher? I'm afraid comparison of patch versions might be tricky and unreliable

Comment (njuettner) [reply service/controller/key/common.go:186]: Agree, I slightly changed it to `return releaseVersion.Major >= v19.Major`. Is this better for you?

Comment (whites11) [reply service/controller/key/common.go:186]: Yes perfect

**Review by whites11 — APPROVED** [review]
Lgtmiiwfy

**Pull request merged**